internal/runner/runner.go

```diff
@@ -71,18 +71,21 @@ func Do(ctx context.Context, opts ...Option) error {
 
 	if isHelp || err != nil {
 		log.Init(log.WithLevel(level.FATAL.String()))
+		defer log.Close()
```

Comment:
I think log.Fatal in main.go may crash if you close log.Object here.

Reply:
I removed them. Is it okay?

Reply:
I think it's okay but we have to close logger, so I think it's not beautiful answer but we should close it after main.go's log.Fatal

Reply:
ah, i got it.

Reply:
oh, I gotta beautiful answer,,,,

Reply:
we should close it in the log.Fatal Function.

Reply:
That's right.

Reply:
log.Fatal kills process we can close it

Reply:
But there may be same thing here, if the runner.Do returns some error?
https://github.com/vdaas/vald/blob/fc1e535f5ed8ac4042fce532d5cbe7746b1fa470/internal/runner/runner.go#L99

Reply:
omg, I make mistale, that's true,,,,

Reply:
hmm, ok, I think current implementation is correct, beacuse if log.Fatal runs we cannot use defer.
I'll approve this PR.

Reply:
i pointed that if runner.Do returns some error, `defer log.Close()` at L99 runner.go will be executed before `log.Fatal` in the cmd/xxx/main.go.
I think the `log.Fatal` may crash.

Reply:
I think it depends on the log library.
glg Close func doesn't do anything, so it doesn't affect log.Fatal panic.
and zap only Syncs to flush the log buffer, so I don't think there is a problem with zap.

Reply:
I see.